src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/MercuryErrorComponentObject.java

```diff
@@ -0,0 +1,28 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.mercury;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.BasePageObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * @ownership Content X-Wing Wikia
+ */
+public class MercuryErrorComponentObject extends BasePageObject {
```

Comment:
It shouldn't be called MercuryError but MercuryAlert.
Those alert can be both successes and errors